api/v1/subschema/common.schema.json

```diff
@@ -82,10 +82,6 @@
               "description": "Overwrite app settings for real device to enable screenshot restriction.",
```

Comment:
The specific appsettings fields for Espresso and XCUITest should be defined separately in https://github.com/saucelabs/saucectl/blob/main/api/v1alpha/framework/espresso.schema.json and https://github.com/saucelabs/saucectl/blob/main/api/v1alpha/framework/xcuitest.schema.json
Mixing these config fields can be confusing in JSON Schema.

Reply:
Makes sense. Moved definitions to relevant files, removing unused ones per platform.